projects/packages/forms/src/util/get-preferred-responses-view.js

```diff
@@ -1,6 +1,5 @@
 import { getJetpackData } from '@automattic/jetpack-shared-extension-utils';
 
 export const PREFERRED_VIEW = window?.jpFormsBlocks?.defaults?.preferredView;
-export const PARTIAL_RESPONSES_PATH =
-	PREFERRED_VIEW === 'classic' ? 'edit.php?post_type=feedback' : 'admin.php?page=jetpack-forms';
+export const PARTIAL_RESPONSES_PATH = 'admin.php?page=jetpack-forms-admin';
```

Comment:
Should we clean up all the `PREFERRED_VIEW` instances? Maybe in a different PR.

Reply:
Sure, there's a lot of things to clean up, but we'll do that on a single pass once we're done with the slug update :)